src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/vedtaksvalidering.ts

```diff
@@ -92,6 +97,12 @@ export const validerUtgiftsperioder = ({
         if (!periodetype) {
             return { ...utgiftsperiodeFeil, periodetype: 'Mangler valg for periodetype' };
         }
+        if (periodetype === EUtgiftsperiodetype.OPPHØR && index === utgiftsperioder.length - 1) {
```

Comment:
Kan lage en const etter `utgiftsperiodeFeil`: 
`const erSistePeriode = index === utgiftsperioder.length - 1`

Også sette inn denne konstanten her:
```suggestion
        if (periodetype === EUtgiftsperiodetype.OPPHØR && erSistePeriode) {
```

Reply:
For bedre lesbarhet :)